pkg/converter/target/externalnode.go

```diff
@@ -77,5 +83,9 @@ func populateExternalNodeFrom(source ExternalNodeSource, externalNode *antreav1a
 			IPs:  ips,
 		})
 	}
-	externalNode.Spec.Interfaces = networkInterface
+	if !reflect.DeepEqual(externalNode.Spec.Interfaces, networkInterface) {
+		externalNode.Spec.Interfaces = networkInterface
```

Comment:
dont you require any sorting on interface before comparison? @archanapholla

Reply:
based on yesterday's discussion, added the sort part